scipeds/data/enums.py

```diff
@@ -48,6 +49,7 @@ class RaceEthn(str, Enum):
     two_or_more = "Two or more races"
     white = "White"
     unknown = "Unknown"
+    total = "Total"
```

Comment:
It might be better to leave race as null rather than create a new category

Reply:
Right, I suppose there isn't a "race ethnicity" column for the df when you group by gender, so there isn't a risk of user confusion.

I'm open to either way! Honestly, whichever way reduces the risk of accidentally double-counting -- which leaving it as Null is probably better

Reply:
Ah, duckdb doesn't allow for NULL values in ENUM columns. So we can either change the race/ethnicity column to a string dtype, or add this "Total" group?

Reply:
There's a slight distinction here - duckdb doesn't allow null values in enum _types_, but null values in a _column with enum values_ are okay. So if we change the query that we use to create the enum _type_ to something like:

```python
create_enum_stmt = f"""
            CREATE TYPE {col}_enum AS ENUM
                (SELECT DISTINCT {col} 
                FROM {COMPLETIONS_TABLE} 
                WHERE {col} IS NOT NULL
                ORDER BY {col})
            """
```

Then we should be able to create the enum type in the way we expect, but still be able to add NULL values to it.

Reply:
Alternatively, we could (accurately) classify them under the existing "unknown" classification.